packages/zendesk-adapter/src/fallback_user/user.ts

```diff
@@ -0,0 +1,140 @@
+  detailedMessage: `The following users are referenced by this instance, but do not exist in the target environment: ${missingUsers.join(', ')}.\nIn addition, we could not get the defined fallback user ${userFallbackValue}. In order to deploy this instance, add these users to your target environment, edit this instance to use valid usernames, or set the target environment's user fallback options.\nLearn more: ${MISSING_USERS_DOC_LINK}`,
+})
+
+const getMissingUsersChangeWarning = (
+  instance: InstanceElement,
+  missingUsers: string[],
+  userFallbackValue: string
+): ChangeError => ({
+  elemID: instance.elemID,
+  severity: 'Warning',
+  message: `${missingUsers.length} usernames will be overridden to ${userFallbackValue}`,
+  detailedMessage: `The following users are referenced by this instance, but do not exist in the target environment: ${missingUsers.join(', ')}.\nIf you continue, they will be set to ${userFallbackValue} according to the environment's user fallback options.\nLearn more: ${MISSING_USERS_DOC_LINK}`,
+})
+
+const getMissingUserPaths = (
+  users: Set<string>,
+  instance: InstanceElement
+): { user: string; path: ElemID }[] => {
+  const userPaths = TYPE_NAME_TO_REPLACER[instance.elemID.typeName]?.(instance)
+  return userPaths.map(path => {
+    const user = resolvePath(instance, path)
+    if (!VALID_USER_VALUES.includes(user) && !users.has(user)) {
+      return { user, path }
+    }
+    return undefined
+  }).filter(values.isDefined)
+}
+
+const getFallbackUserIsMissingUsers = (users: Set<string>) => (
+  instance: InstanceElement
+): { instance: InstanceElement; users: string[] } =>
+  ({ instance, users: _.uniq(getMissingUserPaths(users, instance).map(({ user }) => user)) })
+
+const replaceMissingUsers = (
+  users: Set<string>,
+  fallbackUser: string
+) => (instance: InstanceElement): undefined |
+{ fixedInstance: InstanceElement; missingUsers: string[] } => {
+  const missingUserPaths = getMissingUserPaths(users, instance)
+
+  if (!missingUserPaths) {
+    return undefined
+  }
+  const fixedInstance = instance.clone()
+  missingUserPaths.forEach(({ path }) => setPath(fixedInstance, path, fallbackUser))
+  return { fixedInstance, missingUsers: _.uniq(missingUserPaths.map(({ user }) => user)) }
+}
+
+/**
+ * Change missing users (emails or ids) to fallback user.
+ * If fallback user is not provided, do nothing
+ * The errors returned will vary:
+ * 1. If provided fallback user is valid, return warning severity errors
+ * 2. If provided fallback user is not valid, return error severity errors
+ */
+const missingUsersToFallback: FallBackUserHandler['missingUsersToFallback'] = (
+  client,
+  config
+): FixElementsFunc => async elements => {
+  const paginator = createPaginator({
+    client,
+    paginationFuncCreator: paginate,
+  })
+  const users = await getUsers(paginator)
+  const { defaultMissingUserFallback } = config
+  if (_.isEmpty(users) || defaultMissingUserFallback === undefined) {
```

Comment:
in case someone uses it and sends false instead of {}
```suggestion
  if (_.isEmpty(users) || defaultMissingUserFallback === undefined || defaultMissingUserFallback === false) {
```

Reply:
you are right, my bad :)